devices/Ft6xx6x/samples/Program.cs

```diff
@@ -45,11 +45,64 @@ void TouchInterrupCallback(object sender, PinValueChangedEventArgs pinValueChang
     {
         var point = sensor.GetPoint(true);
         Debug.WriteLine($"ID: {point.TouchId}, X: {point.X}, Y: {point.Y}, Weight: {point.Weigth}, Misc: {point.Miscelaneous}, Event: {point.Event}");
+        discrimitateButtons(point.X, point.Y);
     }
     else if (points == 2)
     {
         var dp = sensor.GetDoublePoints();
         Debug.WriteLine($"ID: {dp.Point1.TouchId}, X: {dp.Point1.X}, Y: {dp.Point1.Y}, Weight: {dp.Point1.Weigth}, Misc: {dp.Point1.Miscelaneous}, Event: {dp.Point1.Event}");
         Debug.WriteLine($"ID: {dp.Point2.TouchId}, X: {dp.Point2.X}, Y: {dp.Point2.Y}, Weight: {dp.Point2.Weigth}, Misc: {dp.Point2.Miscelaneous}, Event: {dp.Point1.Event}");
     }
-}
\ No newline at end of file
+}
+
+static void discrimitateButtons(int x, int y)
```

Comment:
No need for static. 
Method names are camel case, starting with upper case letter.